src/src/com/microsoft/aad/adal/TokenCacheItem.java

```diff
@@ -57,6 +57,8 @@
     private boolean mIsMultiResourceRefreshToken;
 
     private String mTenantId;
+    
+    private String mFamilyClientId;
```

Comment:
Have you checked the upgrade scenario?
If user currently has token without mFamilyClientId will it work fine?